### DISCUSSION TITLE
wmic qfe

### DISCUSSION THREAD

**Discussion by patfla** [microsoft/WSL#6688] [Q&A]
This is neither a bug report nor a feature request.  I was wondering why (on Build 21322) wmic qfe works in a cmd window but not from a powershell (7)?  On powershell7, wmic throws up a "How do you want to open this file" dialog.

I believe that things are in motion in regards to WMI.

**Comment by therealkenc** [comment]
Could not reproduce on 21332 (sic) with pwsh 7.1.0 FWIW.

![image](https://user-images.githubusercontent.com/7049700/111536908-107cfb80-8728-11eb-821f-f81de178e722.png)

**Comment by patfla** [comment]
I didn't know, from powershell7, to do 

`cmd.exe /c ver`

first.

Tried it again.  Run powershell7 and then directly enter 

`wmic qfe`

and I still get "pick an app."  In other words, it's maybe misinterpreting what I want to do.

Or rather, I'm not supposed to directly enter `wmic qfe` at that point because it's not a valid command from powershell7.
